frontend/pages/hosts/details/DeviceUserPage/helpers.ts

```diff
@@ -1,16 +1,6 @@
-import { getErrorReason } from "interfaces/errors";
-
 const DEFAULT_ERROR_MESSAGE = "refetch error.";
 
 // eslint-disable-next-line import/prefer-default-export
 export const getErrorMessage = (e: unknown, hostName: string) => {
-  let errorMessage = getErrorReason(e, {
-    reasonIncludes: "Host does not have MDM turned on",
-  });
-
-  if (!errorMessage) {
-    errorMessage = DEFAULT_ERROR_MESSAGE;
-  }
-
-  return `Host "${hostName}" ${errorMessage}`;
+  return `Host "${hostName}" ${DEFAULT_ERROR_MESSAGE}`;
```

Comment:
Just double checking there isn't supposed to be any more logic in this function?

Reply:
nope. removed it as it was not needed.